source/reference/minio-mc/mc-ilm-rule-ls.rst

```diff
@@ -0,0 +1,150 @@
+.. _minio-mc-ilm-rule-ls:
+
+==================
+``mc ilm rule ls``
+==================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ilm rule ls
+
+.. versionchanged:: RELEASE.2022-12-24T15-21-38Z
+
+   ``mc ilm rule ls`` replaces ``mc ilm ls``.
+
+Syntax
+------
+
+.. start-mc-ilm-rule-ls-desc
+
+The :mc:`mc ilm rule ls` command summarizes all configured object lifecycle management rules on a MinIO bucket in a tabular format.
+
+.. end-mc-ilm-rule-ls-desc
+
+The output of the command might resemble the following:
+
+.. code-block:: shell
```

Comment:
I would either

a) move this under the Example/Syntax section, or
b) move it under behavior as "Output Layout"
c) drop it all together, as outputs like this do change.

Reply:
Moved under the Example tab, which is a better place for it.